src/commands/sendtag.test.ts

```diff
@@ -37,6 +39,6 @@ describe('sendtag', () => {
 	test('returns an array for autocomplete', () => {
 		expect(
 			sendtag.autocomplete?.(context.interaction as unknown as AutocompleteInteraction)
-		).toBeArray();
+		).toBeDefined();
```

Comment:
This is a different test. Better to wrap the result in `Array.isArray` and check that the return of that is `true`.

Reply:
TypeScript already guarantees that this method either returns `Array | undefined`. Therefore, testing that the return value is defined guarantees that it is an array. I utilized type checking like this in many places.

Reply:
Fair enough, makes sense to me.